base/templates/base/base.html

```diff
@@ -105,18 +105,20 @@
                 </div>
             </div>
         {% endif %}
-            <div class="row">
-                <nav class="navbar navbar-default base-navbar base-navbar-bottom">
-                      <div class="container-fluid container-center">
-                        <ul class="nav navbar-nav navbar-right">
-                            <li class="inactive"><a href="/contact/">Contact</a></li>
-                            <li class="inactive"><a href="/legal/impressum/">Imprint</a></li>
-                            <li class="inactive"><a href="/legal/datasec/">Legal</a></li>
-                            <li class="inactive"><a href="/legal/tou/">Terms of Use</a></li>
-                        </ul>
-                      </div>
-                </nav>
-            </div>
+            <nav class="navbar navbar-default  navbar-fixed-bottom">
+                  <div class="container-fluid" style="width:80%; float:left;">
+                    <ul class="nav navbar-nav">
+                        <li class="inactive"><a href="/about/">About us</a></li>
+                        <li class="inactive"><a href="/faq/">FAQ</a></li>
+                        <li class="inactive"><a href="/contact/">Contact us</a></li>
+                        <li class="inactive"><a href="/legal/impressum/">Impressum</a></li>
+                        <li class="inactive"><a href="/legal/datasec/">Datenschutzerklärung</a></li>
+                        <li class="inactive"><a href="/legal/tou/">Terms of use</a></li>
+                        <li class="inactive"><a href="http://opendefinition.org/od/2.0/en/" target="_blank">Open Data {% bootstrap_icon "new-window" %}</a></li>
+                        <li class="inactive"><a href="https://okfn.org/" target="_blank">Open Knowledge  {% bootstrap_icon "new-window" %}</a></li>
+                    </ul>
+                  </div>
+            </nav>
```

Comment:
I think this merge goes in the wrong direction. The deleted part is the one that is on master (and should be)

Reply:
Thanks for spotting that! I kept `develop` version when I should have chosen `HEAD` for the conflict resolution, I did a reset and force pushed the result of the new merge.